src/value.jl

```diff
@@ -143,6 +143,11 @@ function Base.isless(x::CatValue, y::CatValue)
     end
 end
 
+Base.isless(x::CategoricalString, y::AbstractString) = order(x) < order(x.pool[get(x.pool, y)])
+Base.isless(y::AbstractString, x::CategoricalString) = !(isless(x, y) || isequal(x, y))
+Base.isless(x::CategoricalValue{T}, y::T) where {T} = order(x) < order(x.pool[get(x.pool, y)])
```

Comment:
Can we widen `::T` to `::Any`? There's no reason one couldn't compare an `Int64` to an `Int32`. That would also allow using a single method for `CatValue` rather than one method for `CategoricalValue` and one for `CategoricalString`.

Reply:
OK - I will see how it works.